### PR TITLE
MCR-2227 check show permission within iiif request

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2227).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 0b0f9c1ee84dfb9c6809e8ef61ba2cd0930c399b! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5679-pull_request-995

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 38689893f4090e046bdfb1fbc8c83ce126291aae! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5745-pull_request-995

File: mycore-iview2/src/main/resources/xsl/iview2-solrresponse.xsl [hunk 1]
```diff
@@ -109,7 +109,7 @@
       </xsl:variable>
 
       <xsl:if test="$pageToDisplay != ''">
-        <xsl:variable name="object-view-derivate" select="acl:checkPermission($mcrid,'view-derivate')" />
+        <xsl:variable name="object-view-derivate" select="acl:checkPermission($derivate,'show')" />
```
Comment (yagee-de) [inline mycore-iview2/src/main/resources/xsl/iview2-solrresponse.xsl]: permission is 'view' now

**Review by yagee-de — CHANGES_REQUESTED** [review]

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 47b5fb3afd2517f72fe7a6efca8ce3356c64e38a! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5783-pull_request-995

**Review by yagee-de — APPROVED** [review]

**Pull request merged**